main.cpp

```diff
@@ -52,85 +52,86 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:       house
+//  action 1:   the house provides shelter
+house.provideShelter();
+//  action 2:   the house appreciates in value // 
+house.appreciateInValue();
+//  action 3:   the house deteriorates over time
+house.deteriorateOverTime();
 
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       computer  
+//  action 1:   the computer boots up 
+computer.bootUp();
+//  action 2:   the computer overheats
+computer.overheat();
+//  action 3:   the computer shuts down
+computer.shutDown();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       andrew
+//  action 1:   andrew runs errand
+andrew.runErrand();
+//  action 2:   andrew pays cashier
+andrew.payCashier();
+//  action 3:   andrew relaxes
+andrew.relax();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       america
+//  action 1:   america holds election
+america.holdElection();
+//  action 2:   america votes for president
+america.voteForPresident();
+//  action 3:   america placates citizens
+america.placateCitizens();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       camera
+//  action 1:   the camera displays menu
+camera.displayMenu();
+//  action 2:   the camera captures image
+camera.captureImage();
+//  action 3:   the camera exports file
+camera.exportFile();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       julianne
+//  action 1:   julianne wears makeup
+julianne.wearMakeup();
+//  action 2:   julianne selects outfit
+julianne.selectOutfit();
+//  action 3:   julianne decides to excessorize
+julianne.decideToExcessorize();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:       thailand
+//  action 1:   thailand welcomes tourism
+thailand.welcomeTourism();
+//  action 2:   thailand issues temp visa
+thailand.issueTempVisa();
+//  action 3:   thailand profits from tourist
+thailand.profitFromTourist();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.